security/ldap-config-handler/src/main/java/org/codice/ddf/admin/security/ldap/test/LdapTestingCommons.java

```diff
@@ -217,4 +271,24 @@ public LdapConnectionResult result() {
             return result;
         }
     }
+
+    private static boolean configurationExists(LdapConfiguration existingConfiguration,
+            LdapConfiguration newConfiguration) {
+        return existingConfiguration.hostName()
```

Comment:
❓ Should we more aggressively test equality by pinging the two configuration hosts and seeing if they resolve to the same host?